java/src/jmri/beans/VetoableChangeProvider.java

```diff
@@ -2,7 +2,7 @@
 
 import java.beans.VetoableChangeListener;
 import javax.annotation.Nonnull;
-import javax.annotation.Nullable;
+import javax.annotation.CheckForNull;
```

Comment:
In this class, all parameters marked Nullable are done so, because null is accepted, but not checked against. It seems it should be used in this class per https://static.javadoc.io/com.google.code.findbugs/jsr305/3.0.2/javax/annotation/Nullable.html

Nullable is used in the jmri.beans package to annotate parameters that are passed on to java.beans.PropertyChangeSupport which documents that null values are acceptable, so we don't check for them.

Reply:
Both `@Nullable` and `@CheckForNull` are acceptable here, and `@CheckForNull` is preferred.